### PR TITLE
Create Song Information Area

### LINKED ISSUE #17: [Create suggestion flow] Create input song info page (manual flow)

### PR DESCRIPTION
**Pull request by fkoulen**
This will create an area in the New Suggestion flow for users to add song information. 
The required fields are title, artist & motivation. 
Validation is applied.
User can only go to other area once the required fields are filled in.

Tests are supplied as well. 
 (About 300 of the lines are boilercode tests lol)

* Closes #17

### CODE REVIEW

**Review by maarcelh — APPROVED** [review]

File: src/components/new-suggestion/areas/song-information.area.tsx [hunk 1]
```diff
@@ -1,9 +1,162 @@
...
+          )}
+        </div>
+
+        <div className={"input-container"}>
+          <label htmlFor="artist" className="sr-only">
+            Artist(s)
+          </label>
+
+          <div className="input">
+            <input
```
[the remark this answers is not in the document]
Comment (fkoulen) [reply src/components/new-suggestion/areas/song-information.area.tsx:85]: You can just type Justin Bieber ft. David Guetta

[the remark this answers is not in the document]
Comment (fkoulen) [reply src/components/new-suggestion/areas/song-information.area.tsx:85]: But for the Spotify integration it is required to be an array

[the remark this answers is not in the document]
Comment (maarcelh) [reply src/components/new-suggestion/areas/song-information.area.tsx:85]: oke cool

**Comment by maarcelh** [comment]
afbeelding

**Comment by maarcelh** [comment]
Don't we want this border to be yellow?

File: src/components/new-suggestion/areas/review.area.tsx [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 
 const ReviewArea = () => {
+  const newSuggestion = useSelector((state: AppState) => state.newSuggestion.suggestion)
   return (
     <div data-cy="area-review">
       <h2 className={"area-header"}>Review</h2>
+      <p>Title: {newSuggestion.title}</p>
+      <p>Artist(s): {newSuggestion.artist}</p>
+      <p>Motivation: {newSuggestion.motivation}</p>
+      <p>Link: {newSuggestion.link}</p>
```
Comment (maarcelh) [inline src/components/new-suggestion/areas/review.area.tsx:13]: should not show this when link isn't filled in

**Comment by fkoulen** [comment]
> Don't we want this border to be yellow?

Good call! Copied your class names for this hehe

**Pull request merged**